src/protocol.h

```diff
@@ -291,8 +293,13 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 // bits 11..13 are player move type bits (ZQuake extension)
 #define PF_PMC_SHIFT	11
 #define	PF_PMC_MASK		7
+#ifdef FTE_PEXT_TRANS
+#define	PF_ONGROUND		(1<<22)			// ZQuake extension, 14 offset to extra playerflags
```

Comment:
I suggest to use indentation to indicate the parent definition.
These files are containing nothing but ifdefs and defines (more or less). Using indentation makes it easier to read.

```
#ifdef A
# define B
#else
# define C
# ifdef D
#  define E
# endif // D
#endif //A
```

Reply:
@tcsabina as you see here, FTE_PEXT_TRANS will be defined, and then the PF_ONGROUND and PF_SOLID would be offsetted, and the code in ezquake would have to deal with that as it should have from day one. So if this was merged, and ezquake got the new protocol.h file without updated cl_ents.c it would not be able to connect to any server.

Reply:
Good thing is that this part of the alpha support is a trivial change in ezquake once merged in qwprot. Can be dealt with separately from the graphical updates.